### PR TITLE
Chapter 11: Account activation

### PR DESCRIPTION
**Pull request by hoanpl-1064**
Manage User
image

### CODE REVIEW

**Comment by hoanpl-1064** [comment]
ready

File: app/assets/stylesheets/account_activations.css [hunk 1]
```diff
@@ -0,0 +1,4 @@
+/*
```
Comment (vanvtt-0952) [inline app/assets/stylesheets/account_activations.css]: Xóa file

File: app/helpers/account_activations_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module AccountActivationsHelper
```
Comment (vanvtt-0952) [inline app/helpers/account_activations_helper.rb]: xóa file

**Comment by vanvtt-0952** [comment]
merged
fix 2 comment rồi mợt

**Pull request merged**